src/components/dashboard/DashboardPage.jsx

```diff
@@ -80,15 +87,14 @@ const DashboardPage = () => {
           <Tab eventKey="courses" title="Courses">
             {CoursesTabComponent}
           </Tab>
```

Comment:
Unrelated to your specific changes here:
* We may consider only rendering `CoursesTabComponent`, `ProgramListingPage`, and `PathwayProgressListingPage` when the user is actively on the associated tab. As is, we're rendering `ProgramListingPage` and `PathwayProgressListingPage` (which makes API requests and render images) even though the user isn't actively on those tabs. By only rendering the `Tab` content when the tab is active, we may get a bit of a performance bump. This improvement can totally be tackled as a separate chunk of work though :)

Reply:
Definitely +1 on that, will spin this out into its own chunk of work

Leaving open as a reminder